shell/browser/net/proxying_url_loader_factory.cc

```diff
@@ -668,36 +669,19 @@ void ProxyingURLLoaderFactory::InProgressRequest::
       "Non-Authoritative-Reason: WebRequest API\n\n",
       kInternalRedirectStatusCode, redirect_url_.spec().c_str());
 
-  if (factory_->browser_context_->ShouldEnableOutOfBlinkCors()) {
```

Comment:
Should we document that OutofBlinkCORS is enabled by default in the release notes as part of this PR https://www.chromestatus.com/feature/5768642492891136 ?

There is no functionality change but if users face issue with CORS they know where to start looking at.

Reply:
weird according to the status its been enabled by default for a while, guess this check was noop since then. Ignore about documenting it.